docs/interpreter/livy.md

```diff
@@ -85,3 +85,9 @@ You can leverage [Zeppelin Dynamic Form]({{BASE_PATH}}/manual/dynamicform.html).
 %livy.pyspark
 print "${group_by=product_id,product_id|product_name|customer_id|store_id}"
 ```
+
+## Know Issue
+If you are getting an error `Blacklisted configuration values in session config: spark.master`
+
+edit `conf/spark-blacklist.conf` file in livy server and comment out `#spark.master` line.
```

Comment:
I think they are already doing that internally, and its a matter of syncing https://github.com/cloudera/hue/tree/master/apps/spark/java and https://github.com/cloudera/livy. As soon as that is done, this will stop showing up.